docs: Total transactions baseline docs (#5714)
getsentry/sentry-docs: src/docs/product/discover-queries/query-builder.mdx

Each query has a interactive graph that reflects the data presented in the table below. To zoom in, simply click and drag over the area you want to investigate. You can also customize the display and the Y-Axis. This will be saved with your query.

### Total Transactions Display

> Discover queries use sampled transactions, or events-based data. You can compare your query results against the data based on total transactions by enabling the 'Total Transactions' toggle on the chart footer. For example, if you were investigating performance issues on your latest release, you can display `p95(transaction.duration)` for your latest release and the total transactions baseline to visualize how much worse it is than the baseline.
> 
> There are a few reasons why the 'Total Transactions' toggle might be disabled:
> - Selected display is not supported, only available on Total Period display.
> - Selected y-axis is not supported, not available for equations, `apdex()`, `user_misery()` and aggregations performed on fields and tags (for example `count_unique(device.os)`)
> - You are querying both error and transaction data (which is default Discover behaviour), ensure you are querying only transactions data by adding `event.type:transaction` to the query search bar
> - Your project has an outdated SDK which is incompatible with [Dynamic Sampling](/product/data-management-settings/dynamic-sampling/), confirm that your application relies on [supported SDKs](/product/data-management-settings/dynamic-sampling/getting-started/#current-limitations)
> - You have some unparameterized transactions which are incompatible with [Dynamic Sampling](/product/data-management-settings/dynamic-sampling/)

Reviewer: ```suggestion
Discover queries use sampled transactions, or events-based data. You can compare your query results against the data, based on total transactions, by enabling the "Total Transactions" toggle in the chart footer. For example, if you were investigating performance issues in your latest release, you could display `p95(transaction.duration)` for your release and the total transactions baseline to visualize how much worse it was compared to your baseline.

There are a few reasons why the "Total Transactions" toggle might be disabled:
- Selected display is not supported and is only available on "Total Period" display.
- Selected y-axis is not supported, is not available for equations, `apdex()`, `user_misery()`, or aggregations performed on fields and tags (for example `count_unique(device.os)`).
- You're querying both error and transaction data (which is default Discover behavior). To fix this, query transactions data only by adding `event.type:transaction` to the query search bar.
- Your project has an outdated SDK which is incompatible with [Dynamic Sampling](/product/data-management-settings/dynamic-sampling/). To fix this, confirm that your application relies on [supported SDKs](/product/data-management-settings/dynamic-sampling/getting-started/#current-limitations).
- You have some unparameterized transactions which are incompatible with [Dynamic Sampling](/product/data-management-settings/dynamic-sampling/).
```

Reviewer 2: Your edits mostly make sense to me @lizokm! I would make a couple tweaks. This first one because I think "data based on total transactions" is the full noun phrase (please confirm, @shruthilayaj). So, to make it a bit shorter:

> You can compare your query results against total transactions data by enabling the "Total Transactions" toggle in the chart footer.

Or if it feels like it needs to be spelled out for clarity, then:

> You can compare your query results against data based on total transactions by enabling the "Total Transactions" toggle in the chart footer.

I'm pretty sure this needs a comma after transactions:

> You have some unparameterized transactions, which are incompatible with [Dynamic Sampling](/product/data-management-settings/dynamic-sampling/).

This next bit is trickier. I think the edit is probably an improvement, but I'm not actually sure what you're trying to say here @shruthilayaj. Do you mean that the y-axis isn't supported/isn't available or are those first two clauses supposed to work in some other way? Is the rest of list starting at _equations_ a list of all the things that the y-axis isn't supported or available for?

> Selected y-axis is not supported, not available for equations, `apdex()`, `user_misery()` and aggregations performed on fields and tags (for example `count_unique(device.os)`)

Lastly, just the word _Discover_ should always be bolded, but otherwise, these edits look good to me.
